amethyst_input/src/bundle.rs

```diff
@@ -27,7 +27,8 @@ use {Bindings, InputSystem};
 ///
 /// No errors returned from this bundle.
 ///
-#[derive(Default)]
+#[derive(Derivative)]
+#[derivative(Default(bound = "AX: Hash + Eq, AC: Hash + Eq"))]
```

Comment:
Please leave this bound empty, otherwise the bound shows up twice for the `Default` implementation in rustdoc.

Reply:
`bound = ""` is a good way to do this.

Reply:
👍